packages/esm-patient-chart-app/src/actions-buttons/add-past-visit.component.tsx

```diff
@@ -13,29 +15,20 @@ const AddPastVisitOverflowMenuItem: React.FC<AddPastVisitOverflowMenuItemProps>
 }) => {
   const { t } = useTranslation();
 
-  const openModal = useCallback(() => {
+  const handleLaunchModal = useCallback(() => {
```

Comment:
This just renames the event handler to better match recommended [naming conventions](https://react.dev/learn/responding-to-events#adding-event-handlers).